lib/validator/validator_component.cpp

```diff
@@ -1,25 +1,29 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2019-2024 Second State INC
 #include "validator/validator_component.h"
+#include "common/spdlog.h"
 
 #include <variant>
 
-using namespace std::literals;
 
 namespace WasmEdge {
 namespace Validator {
 
+using namespace std::literals;
 using namespace AST::Component;
 
 Expect<void> Validator::validate(const AST::Component::Component &Comp) {
   spdlog::warn("component validation is not done yet."sv);
+  spdlog::info("Component Model Validation is in active development."sv);
```

Comment:
I think @hydai's idea is replace text
```
"component validation is not done yet."
```
with
```
"component validation is in active development."
```

Reply:
Haha. You got me ;-)